backend/src/database/ProductCategory.ts

```diff
@@ -48,4 +49,11 @@ export class ProductCategory extends BaseEntity {
     { nullable: false },
   )
   requisitionConfig: RequisitionConfig;
+
+  @Column({
+    type: "enum",
+    enum: ProductCategoryTyp,
+    nullable: false,
+  })
+  typ: ProductCategoryTyp;
```

Comment:
```suggestion
  type: ProductCategoryType;
```

Reply:
Ja, ich weiß, sieht komisch aus ... bevorzuge auch deine Lösung. Hatte mich aber vorerst an `typ: TextContentTyp` orientiert. Vermutung war, dass Andreas sich etwas dabei gedacht hat nicht `type` zu schreiben, weil das ja auch ein TS-Keyword ist.

Als Attribut ist es aber vermutlich bedenkenlos verwendbar, oder was denkst du?

Reply:
Ich denke ich umschiffe das Problem am besten, indem ich es `kind` oder `categoryType` nenne.

Reply:
Ah, ich verstehe. Daran hab ich nicht gedacht. Üblich ist dann einen Unterstrich anzuhängen: `type_`. Als weitere kurze Alternative, die ich schon mal gesehen hab, finde ich auch `ty` okay. `categoryType` finde ich auch nicht schlecht. `kind` ist auch okay, aber evtl. inkonsistent, weil ja sonst überall von `type` die Rede ist.

Reply:
`TextContentTyp` gibt es ja dann keinen Grund für das fehlende e, oder?

Reply:
TextContentTyp ist denke ich aus Konsistentzgründen so genannt. Würde es aber aktuell nicht ändern, weil das eine (wenn auch banale) Datenbankmigration mit sich bringen würde.

Aktuell schwanke ich, ob ich das `ProductCategory.typ` nicht doch einfach so lasse. So ist es immerhin konsistent mit TextContent.typ. Stattdessen `type_` oder `ty` zu verwenden wäre ja nur eine andere Art von Workaround, welche mit dem in der Codebasis bereits vorhandenen inkonsistent wäre.

`categoryType` ginge auch, hat aber eine gewisse Redundanz.